src/create-matcher.js

```diff
@@ -31,14 +35,15 @@ export function createMatcher (routes: Array<RouteConfig>): Matcher {
 
     if (name) {
       const record = nameMap[name]
+      const paramNames = getParams(record.path)
```

Comment:
We can use [similar code here](https://github.com/znck/vue-router/blob/ab0a779ba6f09f5b28138c9445ce84d7263daef6/src/create-matcher.js#L196-L198) to avoid a separate  getParams function